packages/web-components/src/components/rux-progress/rux-progress.tsx

```diff
@@ -1,5 +1,9 @@
 import { Component, Host, h, Prop, Watch } from '@stencil/core'
 
+/**
+ * @part progress - the HTML progress element of rux-progress
+ * @part output - the HTML output element of rux-progress
```

Comment:
These are kind of odd - you couldn't style these thru a slot but what you can do with a part is still really limited because of how the css is accomplishing the progress bar styling